data/_demodata/applications/datacatalogue/Catalogues.csv

```diff
@@ -1,9 +1,9 @@
-network,type,publisher,name,description
-LifeCycle,project,,LifeCycle,to be filled out
-ATHLETE,project,,ATHLETE,to be filled out
-IPEC,theme,,IPEC,to be filled out
-EUChildNetwork,project,,EUChildNetwork,to be filled out
-OOM,project,,OOM,to be filled out
-LongITools,project,,LongITools,to be filled out
-testNetwork1,theme,,testNetwork1,to be filled out
-testNetworkofNetworks,project,,testNetworkofNetworks,to be filled out
+name,network,type,description,publisher
+LifeCycle,LifeCycle,project,,UMCG
```

Comment:
we should make the catalogue a foundation, because this shouldn't be UMCG.
At least it should be 'MOLGENIS team, UMCG'.